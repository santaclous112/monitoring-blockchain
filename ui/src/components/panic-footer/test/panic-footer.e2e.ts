import { newE2EPage } from '@stencil/core/testing';

describe('panic-footer', () => {
  it('renders', async () => {
    const page = await newE2EPage();
    await page.setContent('<panic-footer></panic-footer>');

    const element = await page.find('panic-footer');
    expect(element).toMatchSnapshot();
  });
});
