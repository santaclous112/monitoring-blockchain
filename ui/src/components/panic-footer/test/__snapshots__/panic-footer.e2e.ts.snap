// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-footer renders 1`] = `
<panic-footer class="hydrated">
  <svc-footer color="primary" headline="Developed by SimplyVC" headline-position="start">
    <svc-buttons-container position="end">
      <svc-button href="https://twitter.com/Simply_VC" icon-name="logo-twitter" target="_blank"></svc-button>
      <svc-button href="https://github.com/SimplyVC" icon-name="logo-github" target="_blank"></svc-button>
      <svc-button href="https://medium.com/simply-vc" icon-name="logo-medium" target="_blank"></svc-button>
      <svc-button href="https://t.me/SimplyVC" icon-src="assets/logos/telegram_logo.svg" target="_blank"></svc-button>
      <svc-button href="https://simply-vc.com.mt" icon-src="assets/logos/simplyvc_logo.svg" target="_blank"></svc-button>
    </svc-buttons-container>
  </svc-footer>
</panic-footer>
`;
